packages/primitives/src/Spinner.tsx

```diff
@@ -56,5 +59,13 @@ export type SpinnerProps = HTMLArkProps<"div"> & JsxStyleProps & SpinnerVariantP
 const StyledSpinner = styled(ark.div, {}, { baseComponent: true });
 
 export const Spinner = forwardRef<HTMLDivElement, SpinnerProps>(({ size, css: cssProp, ...props }, ref) => (
-  <StyledSpinner css={css.raw(spinnerRecipe.raw({ size }), cssProp)} {...props} ref={ref} />
+  <StyledSpinner
+    aria-busy="true"
+    aria-live="polite"
+    aria-valuetext="Loading"
```

Comment:
Bør vel puttes inn i oversettelser.

Reply:
Nei. Vi støtter ikke oversettelser i primitives, og det synes jeg ikke vi skal heller.

Reply:
Denne må vel overstyres alle steder, da hovedspråket på siden er norsk. Da er det like greit å gjøre den required v.h.a typescript.

Reply:
Da liker jeg faktisk oversettelser bedre, ev. under egen gruppering `composites` eller noe, dersom primitives skal være fri for slikt. Jeg mener det ikke er sannsynlig at den tekstverdien kommer til å ende opp med å være forskjellig på alle steder, og da mener jeg det er tungvint å skulle legge den inn hvert sted komponenten brukes hvis det kan unngås (og heller la den være overstyrbar de steder den må overstyres).
Jeg tar den ut og legger den som required for typescript nå (så kan det heller gjøres om senere), men ber om at vi gjør en vurdering av om vi kan håndtere slikt bedre.